src/main/java/com/training/utilities/SamplePage.java

```diff
@@ -0,0 +1,37 @@
+package com.training.utilities;
+
+import com.google.gson.Gson;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class SamplePage {
+    public String getUrl()  {
```

Comment:
This should be generic function where it will retrieve any values for the keys given, you can follow the below prototype:
public String getValue(String key){
  //your code
  return value;
}